docs/resdk-tables.rst

```diff
@@ -179,12 +179,12 @@ can also be ``NaN`` (unknown). Other common case is just the info that
 there is no mutation on a given position. This is a valid information
 also. Given the above, a more realistic example of output is:
 
-=========  =====================  ===================== ========
-sample_id  chr1_123_C>T_Gly11Asp  chr1_126_T>C_Asp12Gly chr1_127
-=========  =====================  ===================== ========
-101        2                      NaN                   0
-102        0                      1                     NaN
-=========  =====================  ===================== ========
+=========  ============  ============ ========
+sample_id  chr1_123_C>T  chr1_126_T>C chr1_127
```

Comment:
Variants like `chr1_127` (only position present without nucleotide change) are not possible anymore since the latest change to `mutations-table`. This kind of variants were reported only in cases where there was no variant present in the sample but was present in the reference file.